contrib/scripts/builder.sh

```diff
@@ -54,6 +54,7 @@ docker run --rm \
 	$MOUNT_GOCACHE_DIR \
 	$MOUNT_GOMODCACHE_DIR \
 	$MOUNT_CCACHE_DIR \
+	-v /sys:/sys \
```

Comment:
Why is this necessary?

Reply:
To be able to access the trace_pipe to recover the pkt dumps:

* https://github.com/cilium/cilium/pull/40294/commits/563d42af7b1ca86d281a162520c1d6cef801c42b#diff-d842ee12721dfe61b336fa06a6646dd78f2f733e8000c2d014fc01ff2cd5c0a8R103

Now that you mention it, it might be a good idea to move it to `DOCKER_ARGS` in the `run_bpf_tests` `Makefile`'s target